src/Resources/app/administration/src/module/sw-dashboard/component/sw-dashboard-statistics-promotion-card/index.js

```diff
@@ -0,0 +1,98 @@
+import template from './sw-dashboard-statistics-promotion-card.html.twig';
+import './sw-dashboard-statistics-promotion-card.scss';
+
+const STATISTICS_APP_NAME = 'StatisticsService';
+const BADGE_NEW_REMOVAL_DATE = '2025-01-01 00:00:00.000';
+
+export default Shopware.Component.wrapComponentConfig({
+    template,
+
+    inject: ['extensionStoreDataService', 'acl'],
+
+    i18n: {
+        messages: {
+            /* eslint-disable max-len */
+            'en-GB': {
+                'badge-new': 'Available now',
+                'promotion-title': 'Shopware Analytics',
+                'promotion-text': 'Discover the new suite of KPIs with your sales and performance metrics in Shopware. Get ready to benefit from an ever-expanding range of fresh insights to support your journey toward success!',
+                'app-name': 'Shopware Analytics',
+                'app-description': 'Unlock store performance metrics',
+                'go-to-app': 'Try it out now'
+            },
+            'de-DE': {
+                'badge-new': 'Jetzt verfügbar',
+                'promotion-title': 'Shopware Analytics',
+                'promotion-text': 'Entdecke die neue Suite an Kennzahlen mit Deinen Verkaufs- und Leistungsmetriken in Shopware. Mach Dich bereit, von einer ständig wachsenden Auswahl an Auswertungen zu profitieren, die Deine Reise zum Erfolg unterstützen!',
```

Comment:
Should we also use "KPIs" instead of "Kennzahlen"? Kennzahlen sounds so 1960's :smile: